common/test/common_tests/helpers/files.cljc

```diff
@@ -131,6 +135,8 @@
         shape     (sample-shape label (dissoc params :parent-label))
         parent-id (when parent-label
                     (thi/id parent-label))
+        _         (when parent-label
+                    (t/is (some? parent-id)))
```

Comment:
I wouldn't add checks here. We can assume that the test setup is well done. We are not testing the test itself.